api/Element.json

```diff
@@ -7845,6 +7943,178 @@
             "deprecated": false
           }
         }
+      },
+      "webkitmouseforcechanged_event": {
+        "__compat": {
+          "description": "<code>webkitmouseforcechanged</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/webkitmouseforcechanged_event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
```

Comment:
`"webview_android"` is missing from this and several others. It should match `"chrome"` (at least for this API). Please add it throughout.

Reply:
IMO this should be done in a follow-up PR.

Reply:
I have made the update and pushed it.